### PR TITLE
Inline Case Search Test Cases

### PR DESCRIPTION
**Pull request by kiran-tripathy**
## Summary
Added Case Search Workflows for Inline Search

## Description

[Inline Case Search tests in Case Search Regression Plan](https://docs.google.com/spreadsheets/d/1sPLEAJFlhztZ656TH4jk-tjUbhu_YE-8dvVdgiy3xQQ/edit#gid=894360002)

### Link to Jira Ticket (if applicable)
[https://dimagi-dev.atlassian.net/browse/QA-4182](https://dimagi-dev.atlassian.net/browse/QA-4182)

## QA Checklist

- [x] Label(s) and Assignee added
- [x] PR sent for review
- [ ] Documentation (if applicable) added/updated

### CODE REVIEW

File: Features/CaseSearch/test_pages/casesearch_page.py [hunk 1]
```diff
@@ -7,7 +7,8 @@
 from selenium.webdriver.common.by import By
 
 from common_utilities.selenium.base_page import BasePage
-from Features.CaseSearch.constants import TEXT_INPUT, COMBOBOX, YES, NO, text, combobox, HOME, WORK, PREV_MENU, MENU, FIRST_MENU, HOME_SCREEN
+from Features.CaseSearch.constants import TEXT_INPUT, COMBOBOX, YES, NO, text, combobox, HOME, WORK, PREV_MENU, MENU, \
```
Comment (kbo001) [inline Features/CaseSearch/test_pages/casesearch_page.py:10]: try to import Features.CaseSearch.constants as a whole instead of importing each one.

**Review by kbo001 — CHANGES_REQUESTED** [review]

File: Features/CaseSearch/test_pages/casesearch_page.py (hunk 1 above)
Comment (kiran-tripathy) [reply Features/CaseSearch/test_pages/casesearch_page.py:10]: I can import all using `Features.CaseSearch.constants import *`, but is there anything concerning with this? @kbo001

Comment (kbo001) [reply Features/CaseSearch/test_pages/casesearch_page.py:10]: No, nothing concerning. It just the import looks really long. So you can simply write `import Features.CaseSearch.constants`. If it gives any errors then just go with the original. Other than that all looks good to me.

Comment (kiran-tripathy) [reply Features/CaseSearch/test_pages/casesearch_page.py]: ` import Features.CaseSearch.constants` doesn't work, so added `from Features.CaseSearch.constants import *` in
https://github.com/dimagi/dimagi-qa/pull/171/commits/4419835dbb442ad22e320b577dda7514b1b1a1d9

**Review by kbo001 — APPROVED** [review]

**Pull request merged**